manifests/ima.pp

```diff
@@ -32,15 +35,16 @@
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class tpm::ima (
-  Boolean              $enable        = true,
-  Boolean              $manage_policy = false,
-  Stdlib::AbsolutePath $mount_dir     = '/sys/kernel/security',
-  Boolean              $ima_audit     = true,
-  Tpm::Ima::Template   $ima_template  = 'ima-ng',
-  String               $ima_hash      = 'sha256',
-  Boolean              $ima_tcb       = true,
-  Integer              $log_max_size  = 30000000
-){
+  Boolean                        $enable                  = true,
+  Boolean                        $manage_appraise         = false,
+  Boolean                        $manage_policy           = false,
+  Stdlib::AbsolutePath           $mount_dir               = '/sys/kernel/security',
+  Boolean                        $ima_tcb                 = true,
+  Boolean                        $ima_audit               = false,
+  Tpm::Ima::Template             $ima_template            = 'ima-ng',
+  String                         $ima_hash                = 'sha256',
```

Comment:
Are there a limited set of these? If so, should we enumerate them?

If not, we should probably at least make this `String[1]`